### PR TITLE
Add Spanish translation for the Terms of Service

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add Spanish translation for the [Terms of Service](https://github.com/liberapay/liberapay.com/tree/master/i18n/terms).

### CODE REVIEW

File: i18n/terms/es.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Términos del servicio Liberapay.
+
+Este contrato es entre Liberapay, una organización sin fines de lucro fundada en 2015
+basado en la legislación francesa, en lo sucesivo denominado "la organización", y el
+persona que se inscribe en su servicio homónimo en lo sucesivo conocido como "el
```
Comment (jorgesumle) [inline i18n/terms/es.md:5]: la persona

**Review by jorgesumle — COMMENTED** [review]
The translation is not the greatest, maybe I can review it later.

**Comment by Panquesito7** [comment]
@trebmuh, would you mind reviewing this PR? Thanks. 🙂

**Comment by trebmuh** [comment]
Unfortunately, I don't speak Spanish.

**Review by jorgesumle — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
All mentioned issues should be fixed now. Thanks for your review and comments. 🙂

**Comment by jorgesumle** [comment]
> All mentioned issues should be fixed now. Thanks for your review and comments. slightly_smiling_face

Not really, it should say «perderán» (*con tilde en la a*), you haven't fixed that.

**Comment by Panquesito7** [comment]
> it should say «perderán»

Fixed now. Thanks.

**Pull request merged**